yarn-project/end-to-end/src/e2e_key_registry.test.ts

```diff
@@ -103,14 +111,15 @@ describe('Key Registry', () => {
     it('registers', async () => {
       await keyRegistry
         .withWallet(wallets[0])
-        .methods.register(
-          account,
-          account.partialAddress,
-          account.masterNullifierPublicKey,
-          account.masterIncomingViewingPublicKey,
-          account.masterOutgoingViewingPublicKey,
-          account.masterTaggingPublicKey,
-        )
+        .methods.register(account, account.partialAddress, {
```

Comment:
Think in some of the e2e we had a `toNoirStruct` but better handle the actual problem of the encoder.